src/App/Fossa/FossaAPIV1.hs

```diff
@@ -64,7 +55,7 @@ import Data.Text (Text)
 import Data.Text qualified as Text
 import Data.Word (Word8)
 import Effect.Logger
-import Fossa.API.Types (ApiOpts, ArchiveComponents, Issues, SignedURL, signedURL, useApiOpts)
+import Fossa.API.Types
```

Comment:
Don't use implicit imports:
https://github.com/fossas/fossa-cli/blob/master/docs/contributing/STYLE-GUIDE.md#dont-use-implicit-blanket-imports

We should also clean up the pre-existing implicit imports here, if you're using VSCode there's a code action for making imports explicit automatically (you'll have to run the formatter afterward).

Reply:
Also, we should probably remove the prelude import, it does nothing.